gallery/src/styles/NxList/NxListPage.tsx

```diff
@@ -129,6 +140,7 @@ const NxListPage = () =>
             </NxTableRow>
             <NxTableRow>
               <NxTableCell><code className="nx-code">.nx-list__item</code></NxTableCell>
```

Comment:
Tedious I know, but when Master gets merged into this branch you'll need to update the `.nx-list__item` entry in the top table.

Reply:
> Tedious I know, but when Master gets merged into this branch you'll need to update the .nx-list__item entry in the top table.

f31081e3